kernel/arch/dreamcast/hardware/spu.c

```diff
@@ -62,6 +64,38 @@ void spu_memload(uint32 dst, void *src_void, int length) {
     }
 }
 
+void spu_memload_sq(uint32 dst, void *src_void, int length) {
+    uint8 *src = (uint8 *)src_void;
```

Comment:
Please keep variable declarations at the beginning of a scope.

```suggestion
    uint8 *src = (uint8 *)src_void;
    int aligned_len, old;
```